torch/csrc/sparse/cusparseLt.h

```diff
@@ -0,0 +1,367 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Is vendoring this the solution recommended from nvidia? Isn't this available from cuda headers?

Reply:
Yes, I think this might actually be unnecessary